File: 12 - Runners/12 - Runners/Food.swift
//
//  Food.swift
//  12 - Runners
//
//  Created by Minura Iddamalgoda on 13/2/17.
//  Copyright © 2017 Minura Iddamalgoda. All rights reserved.
//

import Foundation
import UIKit

class Food : NSObject {
    var name: String = ""
    var type: FoodType
    var price: Float = 0.0
    var stockRemaining: Int = 0
    var image: UIImage
    
    init(_ name: String, _ type: FoodType, _ price: Float, _ stockRemaining: Int, _ image: UIImage) {
        self.name = name
        self.type = type
        self.price = price
        self.stockRemaining = stockRemaining
        self.image = image
    }
    
    enum FoodType {
        case Steak, Chicken
    }
}

File: 11-Runners/app/src/main/java/com/iddamal/min/a11_runners/Food.java
package com.iddamal.min.a11_runners;

import android.graphics.drawable.Drawable;

public class Food {
    private String mName;
    private float mPrice;
    private FoodType mType;
    private Drawable mImage;
    private int mStockRemaining;

    private enum FoodType {
        STEAK, CHICKEN;
    }

    public Food(String name, float price, FoodType type, Drawable image, int stockRemaining) {
        mName = name;
        mPrice = price;
        mType = type;
        mImage = image;
        mStockRemaining = stockRemaining;
    }
}
